business_register/converter/pep.py

```diff
@@ -1,10 +1,385 @@
+                        already_stored = True
+                        # update_fields = []
+                        # if stored_person.relationship_type_eng != relationship_type_eng:
+                        #     stored_person.relationship_type_eng = relationship_type_eng
+                        #     update_fields.append('relationship_type_eng')
+                        # if stored_person.is_pep != is_pep:
+                        #     stored_person.is_pep = is_pep
+                        #     update_fields.append('is_pep')
+                        # if stored_person.start_date != start_date:
+                        #     stored_person.start_date = start_date
+                        #     update_fields.append('start_date')
+                        # if stored_person.confirmation_date != confirmation_date:
+                        #     stored_person.confirmation_date = confirmation_date
+                        #     update_fields.append('confirmation_date')
+                        # if stored_person.end_date != end_date:
+                        #     stored_person.end_date = end_date
+                        #     update_fields.append('end_date')
+                        # if len(update_fields):
+                        #     stored_person.save(update_fields=update_fields)
+                        break
+            if not already_stored:
+                PepRelatedPerson.objects.create(
+                    pep=pep,
+                    fullname=fullname,
+                    fullname_eng=fullname_eng,
+                    relationship_type=relationship_type,
+                    relationship_type_eng=relationship_type_eng,
+                    is_pep=is_pep,
+                    start_date=start_date,
+                    confirmation_date=confirmation_date,
+                    end_date=end_date
+                )
+
     def save_to_db(self, json_file):
         data = self.load_json(json_file)
-        for pep in data:
-            print(pep.keys())
-            break
+        for pep_dict in data:
+            fullname = pep_dict['full_name'].lower()
+            fullname_eng = pep_dict['full_name_en'].lower()
+            fullname_transcriptions_eng = pep_dict['names'].lower()
+            last_job_title = None
+            if 'last_job_title' in pep_dict:
+                last_job_title = pep_dict['last_job_title'].lower()
+            last_job_title_eng = None
+            if 'last_job_title_en' in pep_dict:
+                last_job_title_eng = pep_dict['last_job_title_en'].lower()
+            last_employer = None
+            if 'last_workplace' in pep_dict:
+                last_employer = pep_dict['last_workplace'].lower()
+            last_employer_eng = None
+            if 'last_workplace_en' in pep_dict:
+                last_employer_eng = pep_dict['last_workplace_en'].lower()
+            info = None
+            if 'wiki_uk' in pep_dict:
+                info = pep_dict['wiki_uk']
```

Comment:
this and all subsequent and similar blocks must be rewritten in a different format
```suggestion
            info = pep_dict.get('wiki_uk', None)
```

Reply:
implemented, good idea!